go/test/endtoend/vtgate/queries/timeout/timeout_test.go

```diff
@@ -179,3 +179,30 @@ func TestQueryTimeoutWithoutVTGateDefault(t *testing.T) {
 	_, err = utils.ExecAllowError(t, mcmp.VtConn, "select sleep(5) from dual")
 	assert.Error(t, err)
 }
+
+// TestOverallQueryTimeout tests that the query timeout is applied to the overall execution of a query
+// and not just individual routes.
+func TestOverallQueryTimeout(t *testing.T) {
+	utils.SkipIfBinaryIsBelowVersion(t, 21, "vtgate")
```

Comment:
Since we changed our downgrade tests, I don't think we need to include the skip if binary is below version command.

Reply:
We would still need this. For upgrade tests running in a PR against release-20.0, we would end up using the code in `main` (upgrade target) to run the tests with both `vtgate N+1, vttablet N` and `vtgate N, vttablet N+1`. For the latter case we don't want to run this test. So, we need this skip line.